### PR TITLE
Add tip for handling "unable to login" issue.

### PR DESCRIPTION
**Pull request by hfloyd**

### CODE REVIEW

**Comment by dampee** [comment]
@hfloyd Thanks for your contribution. We'll review this and come back to this soon.

File: Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md [hunk 1]
```diff
@@ -37,6 +37,12 @@ Should something fail during the migration, the Development environment can alwa
 * Run the Umbraco 8 project locally
 * The migration will need to be authorized - Cloud credentials is used for this
 
+:::note
+If your login doesn't seem to be working, copy the attributes from your v7 web.config "UsersMembershipProvider" to the v8 web.config and try again:
+
+       <add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="8" useLegacyEncoding="true" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" passwordFormat="Hashed" />
+
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md:44]: Making sure the note is closed.
```suggestion
:::
```

File: Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md [hunk 2]
```diff
@@ -37,6 +37,12 @@ Should something fail during the migration, the Development environment can alwa
 * Run the Umbraco 8 project locally
 * The migration will need to be authorized - Cloud credentials is used for this
 
+:::note
+If your login doesn't seem to be working, copy the attributes from your v7 web.config "UsersMembershipProvider" to the v8 web.config and try again:
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md:41]:
```suggestion
If your login doesn't seem to be working, try the following approach:

* Copy the `UsersMembershipProvider` attributes from your Umbraco 7 web.config, to the Umbraco 8 web.config
* Try again

Below is an example of how the attribute can look.
```

File: Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md [hunk 3]
```diff
@@ -37,6 +37,12 @@ Should something fail during the migration, the Development environment can alwa
...
+
+       <add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="8" useLegacyEncoding="true" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" passwordFormat="Hashed" />
```
Comment (sofietoft) [inline Umbraco-Cloud/Upgrades/Migrating-from-7-to-8/index.md:43]:
```suggestion
```xml
<add name="UsersMembershipProvider" type="Umbraco.Web.Security.Providers.UsersMembershipProvider, Umbraco" minRequiredNonalphanumericCharacters="0" minRequiredPasswordLength="8" useLegacyEncoding="true" enablePasswordRetrieval="false" enablePasswordReset="true" requiresQuestionAndAnswer="false" passwordFormat="Hashed" />
```

```

**Comment by hfloyd** [comment]
@sofietoft Take a look now.

**Comment by sofietoft** [comment]
Looks great @hfloyd ! 😄 

Thanks a lot for putting in the work to get this added to the docs 💪 
Much appreciated. Let's get it merged then 😁

**Pull request merged**